.github/workflows/Automation Master Legacy Workflow.yml

```diff
@@ -279,40 +279,125 @@ jobs:
             exit(1)
         shell: python
 
+  artifact_creation:
+      name: Artifact Creation
+      runs-on: ubuntu-latest
+      steps:
+        # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+        - uses: actions/checkout@v4
+          with:
+            fetch-depth: 0
```

Comment:
Fetch depth is not necessary for this job as the branch logic & SonarCloud are not part of this job.